fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -3780,23 +3781,25 @@ private Bundle createSearchBundle(List<Resource> resources, FHIRSearchContext se
 
         Bundle.Builder bundleBuider =
                 Bundle.builder().type(BundleType.SEARCHSET).id(Id.of(UUID.randomUUID().toString())).total(com.ibm.fhir.model.type.UnsignedInt.of((int) searchContext.getTotalCount()));
-
-        for (Resource resource : resources) {
-            if (resource.getId() == null || !resource.getId().hasValue()) {
-                throw new IllegalStateException("Returned resources must have an id.");
+ //       if (!searchContext.getSummaryParameter().contentEquals(SearchConstants.SUMMARY_COUNT)) {
```

Comment:
why is this commented out?  can it be removed?

Reply:
good catch, this was one of the solution which skips the entries for summary count. and it's not needed anymore.

Reply:
done!